ouroboros-network-framework/src/Ouroboros/Network/InboundGovernor.hs

```diff
@@ -155,7 +173,17 @@ inboundGovernor trTracer tracer inboundInfoChannel
                  )
                  (igsConnections state)
             <> FirstToFinish (
-                 NewConnection <$> InfoChannel.readMessage inboundInfoChannel)
+                 NewConnection <$> InfoChannel.readMessage inboundInfoChannel
+               )
+            <> FirstToFinish  (
+                 -- move connections from
```

Comment:
unfinished comment?

Reply:
hmm, maybe it was cut during a rebase, or just unfinished.